### ISSUE TITLE
Cannot upgrade ReaPack

### ISSUE THREAD

**Issue by przemoc** [cfillion/reapack#19]
I'm using ReaPack v1.1rc3.

### ReaPack: Transaction Report
All done! Description of the changes:
~~~
0 installed packages, 0 updates, 0 removed files and 1 error

========== Errors: ==========

UserPlugins\reaper_reapack64.dll:
  Cannot rename to target: File exists
~~~

**Comment by cfillion** [comment]
I'm assuming you were using v1.1rc2 or v1.1rc3 before. These two versions did not correctly delete `ReaPack/old_reaper_reapack64.dll.tmp` from the resource directory after an update. Remove that file manually, and the update should go well.

**Comment by cfillion** [comment]
v1.1rc2 and v1.1rc3 did not correctly delete `ReaPack/old_reaper_reapack64.dll.tmp` from the resource directory on startup. Remove that file manually, and the update should go well. That bug was fixed in v1.1rc4.

**Comment by przemoc** [comment]
I removed `%APPDATA%/REAPER/ReaPack/old_reaper_reapack64.dll.tmp` and it helped indeed. Thanks!
